www/include/RunResultHtmlTable.php

```diff
@@ -17,7 +17,7 @@ class RunResultHtmlTable {
   const COL_RESULT = "result";
   const COL_COST = "cost";
   const COL_CERTIFICATE_BYTES = "certificate_bytes";
-  const COL_FIRST_CONTENTFUL_PAINT = 'chromeUserTiming.firstContentfulPaint';
+  const COL_FIRST_CONTENTFUL_PAINT = 'PerformancePaintTiming.first-contentful-paint';
```

Comment:
If we switch the column we will break it for existing tests.  We should probably see if the logic that uses it can "prefer" a new value but fall back to the previous one if the new one isn't there.

Another option would be to have page_data.inc monkey-patch the value into the new location automatically.

I'd also prefer to make it a top-level "FirstContentfulPaint" and then pull it from the most accurate underlying source available (chromeUserTiming trace events will be more accurate than the in-page FCP since it accounts for JS redirects, etc).

Reply:
> (chromeUserTiming trace events will be more accurate than the in-page FCP since it accounts for JS redirects, etc)

Was wondering that! Ok. Good call.